lib/huginn_agent/templates/newagent/spec/spec_helper.rb.tt

```diff
@@ -0,0 +1,7 @@
+base_path = File.expand_path('../../', __FILE__)
```

Comment:
@dsander determining the absolute path of the huginn_agent is necessary to run the spec outside of `spec_runner.rb`, e.g.
```bash
$ pwd
my_agent
$ (cd spec/huginn; bundle exec rspec --require ../spec_helper.rb ../*_spec.rb)
```
while `rake spec` is a convenient helper, when you try to set a breakpoint with `byebug`, the `popen3`/[`shell_out`](https://github.com/cantino/huginn_agent/blob/f4edd3f001ab85d6519ba7800bd66f3d878b5698/lib/huginn_agent/spec_runner.rb#L46) caused some STDIN inference.